scripts/update_op_coverage.py

```diff
@@ -62,7 +63,8 @@ def get_backend_op_pair(test):
   test_suite_backends = {
       'check_vmla_vmla': 'vmla',
       'check_llvm-ir_llvm': 'llvm-ir',
-      'check_vulkan-spirv_vulkan': 'vulkan-spirv'
+      'check_vulkan-spirv_vulkan': 'vulkan-spirv',
```

Comment:
How is the table formatting with an extra column? I thought there were some style issues with making it too wide.